app/subscriber.py

```diff
@@ -20,7 +20,7 @@
 
 def receipt_to_case(message: Message):
```

Comment:
Could maybe change name of this function to something like `send_receipt_message` now instead?